Kuinox.NupkgDeterministicator/Program.cs

```diff
@@ -18,30 +19,51 @@ public class Build
 
     public static int Main(string[] args)
     {
-        if (args.Length == 0)
+        // todo: a directory path should imply all package files (*.nupkg, *.snupkg) in the directory should be determinized
```

Comment:
Why not keeping a single path as input and have the caller do the job of iterating instead ?

Reply:
I had forgotten MSBuild has [batching](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-batching?view=vs-2022) for iteration.